Algorithms/Sorting/Bubble_Sort.cpp

```diff
@@ -0,0 +1,56 @@
+//Name : Shivam Singhal
+//Email ID : shivamsinghal0610@gmail.com
+
+#include <bits/stdc++.h>
+using namespace std;
+
+void bubbleSort(long arr[], long n)
+{
+  long flag = 0;
+  for(long i=0; i<(n-1); i++){
+    flag = 0;
+    for(long j=0; j<(n-i-1) ;j++)
+    {
+      if(arr[j]>arr[j+1]){
+        ll t = arr[j];
+        arr[j] = arr[j+1];
+        arr[j+1] = t;
+        flag = 1;
+      }
+    }
+    //checks if all elements are sorted or not
+    if(flag==0) 
+      break;
+  }
+}
+ 
+void printArray(long arr[],long size){
+    for(long i=0; i<size; i++)
+      printf("%lld ", arr[i]);
+    printf("\n");
+}
+ 
+// Driver program to test above functions
+int main(){
+    long n;
+    scanf("%lld",&n);
```

Comment:
Could you please provide prompts to the user for the input like "Please enter the count of numbers:" ? 
Please do so in all programs.